nautobot/ipam/querysets.py

```diff
@@ -474,3 +475,7 @@ def filter_address_or_pk_in(self, addresses, pk_values=None):
             q |= Q(pk__in=pk_values)
 
         return super().filter(q)
+
+
+class VLANQuerySet(LocationFieldReplaceMixin, BaseNetworkQuerySet):
```

Comment:
I don't *think* `VLANQuerySet` should inherit from `BaseNetworkQuerySet` - isn't that just for Prefix/IPAddress usage? Might just need to be a `RestrictedQuerySet` subclass instead?